nifi-nar-bundles/nifi-slack-bundle/nifi-slack-processors/src/main/java/org/apache/nifi/processors/slack/PutSlack.java

```diff
@@ -222,7 +273,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                 session.transfer(flowFile, REL_FAILURE);
                 context.yield();
             }
-        } catch (IOException e) {
+        } catch (IOException | JsonParsingException e) {
```

Comment:
This seems like a different error. Even if the behavior (penalize and route to failure) is the same, we probably should have two separate logging messages, to indicate the difference between a connection error vs a "bad data" error.